tests/async_get_result.cpp

```diff
@@ -35,399 +36,414 @@ using ozo::impl::query_state;
 using namespace testing;
 using ozo::error_code;
 
-GTEST("ozo::async_get_result_op::perform()") {
+struct async_get_result_op : testing::Test {
```

Comment:
Why do you both use `using namespace testing` and explicit name for this namespace?

Reply:
Just a habit. Any problem with it? Should I remove it?

Reply:
The problem is one will have one more question "Why? Is there any hidden purpose?". Remove one of these.

Reply:
Ok, **done**